views/application.pug

```diff
@@ -10,5 +10,11 @@ html
           a(class="navbar-text" href="/users/new") NEW USER
         li.nav-item
           a(class="navbar-text" href="/search") SEARCH
-
+        li.nav-item
+          button(class="btn btn-success like-button") LIKE THE APP
     block content
```

Comment:
this is the reference for actual template content area. Application.pug is a template that gets extended with our actual page templates. Actual page template start at the block content. Check: https://pugjs.org/language/extends.html